compiler/circle-quantizer/src/CircleQuantizer.cpp

```diff
@@ -125,6 +126,15 @@ int entry(int argc, char **argv)
           "Three arguments required: tensor_name(string), "
           "scale(float) zero_point(int)");
 
+  arser.add_argument(cq)
+    .nargs(2)
+    .type(arser::DataType::STR_VEC)
+    .required(false)
+    .accumulated(true)
+    .help("Copy quantization parameter from a tensor to another tensor."
```

Comment:
Do we usually have several tensors to apply this or can there be like hundreds of tensors?
If there are lots of tensor pairs, we can provide an external text file with form of 'a tensor, another tensor' 
```
t1,t11
t2,t21,
...
tn,tn1
```

Reply:
Currently our use case is copying the qparam of a single tensor. I think the number of target tensors will not be too large, because fort that case it would be better to requantize the model.

I agree that the text file interface is useful for designating many tensors. Let's revisit the interface when we receive a new request from our customer.